### PR TITLE
fix: add log rotation for CIS log file

### PR DESCRIPTION
**Pull request by phealy**
**What type of PR is this?**
/kind bug

**What this PR does / why we need it**:
/etc/rsyslog.d/60-CIS.conf adds logging for local messages to /var/log/localmessages, but nothing rotates this log file so it can fill the disk on long-lived nodes. This PR adds log rotation for this file.

**Which issue(s) this PR fixes**:

**Requirements**:

- [x] uses [conventional commit messages](https://www.conventionalcommits.org/)
  
- [ ] includes documentation
- [ ] adds unit tests
- [ ] tested upgrade from previous version

**Special notes for your reviewer**:

**Release note**:

```
/var/log/localmessages will now be rotated with the same configuration as /var/log/messages.
```

### CODE REVIEW

File: parts/linux/cloud-init/artifacts/logrotate-d-rsyslog-CIS.conf [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+  rotate 5
+  daily
+  maxsize 300M
+  missingok
+  notifempty
+  compress
+  delaycompress
+  sharedscripts
+  postrotate
+      systemctl kill -s HUP rsyslog.service
```
Comment (djsly) [inline parts/linux/cloud-init/artifacts/logrotate-d-rsyslog-CIS.conf:14]: what is this for ?

**Review by djsly — COMMENTED** [review]
I think you need to run `make generate` ?

**Review by djsly — APPROVED** [review]

**Comment by phealy** [comment]
> I think you need to run `make generate` ?

No changes to CSE or cloud-init, so nothing for generate to do (I did run make generate before committing but there were no changes).

**Comment by UtheMan** [comment]
Question - what sort of things are we logging there? Is 5 days of log history enough in case of issues?

**Review by UtheMan — APPROVED** [review]

**Comment by phealy** [comment]
Tested both ubuntu and mariner VHDs to confirm functionality.

**Pull request merged**